pkg/flowkit/config/json/network_test.go

```diff
@@ -26,7 +26,10 @@ import (
 
 func Test_ConfigNetworkSimple(t *testing.T) {
 	b := []byte(`{
-    "testnet": "access.testnet.nodes.onflow.org:9000"
+    "testnet": {
+			"host": "access.testnet.nodes.onflow.org:9000",
+			"network-key": ""
```

Comment:
Could we add a test case for if we're using the complex format but `network-key` is fully omitted?
```
"a-network": {
     "host": "access.testnet.nodes.onflow.org:9000"
}
```

Reply:
https://github.com/onflow/flow-cli/pull/449/commits/0e29e14541a6f2d948b2357a9b7ff9df72b5330b

Reply:
this format btw should be converted to `"a-netowrk": "access.testnet.nodes.onflow.org:9000"` since if you are only providing data needed by simple schema you should actually provide simple schema.